tests/aliases.tftest.hcl

```diff
@@ -0,0 +1,103 @@
+variables {
+  resource_name = "opslevel_alias"
+  # required fields
```

Comment:
`aliases = toset(["one", "two", "three"])` could be declared once up here and referenced everywhere below

Reply:
oh sorry i didn't respond well to this one.  I didn't go with a top level vars because each run statement had a different `aliases` list.